packages/cli/src/databases/migrations/mysqldb/1693554410387-DisallowOrphanExecutions.ts

```diff
@@ -0,0 +1,26 @@
+import type { MigrationContext, ReversibleMigration } from '@db/types';
+
+export class DisallowOrphanExecutions1693554410387 implements ReversibleMigration {
+	/**
+	 * Ensure all executions point to a workflow.
+	 */
+	async up({ queryRunner, escape }: MigrationContext) {
+		const executionEntity = escape.tableName('execution_entity');
```

Comment:
why not combine the 3 migrations into a single one?

Reply:
How do you mean? The syntax to modify a column differs between MySQL and Postgres, and sqlite requires recreating the table, and it looks like the DSL does not support this yet.